src/actions/index.jsx

```diff
@@ -1,22 +1,32 @@
-
+export const DELETE = 'DELETE';
 export const FILTER = 'FILTER';
-export const DELETE = 'DELETE'
+// export const ADD = 'ADD';
+
+export const deleteItem = (list) => ({ type: DELETE, payload: list });
```

Comment:
You use different syntax(arrow - 5 | function - 7, 14) for action creators, why? Choose which one to use